.github/workflows/production.yml

```diff
@@ -45,6 +45,7 @@ jobs:
           team: ${{secrets.HEROKU_TEAM}}
           appdir: server
         env:
+          HD_FEATURE_FLAG_SELF_SERVICE: ${{ vars.FEATURE_FLAG_SELF_SERVICE }}
```

Comment:
Is this environment variable meant to be set in `development.env`?

Reply:
It depends. We use the same `FEATURE_FLAG_SELF_SERVICE` variable in both the client and server. When running locally, the server's instance of `FEATURE_FLAG_SELF_SERVICE` is set in `server/development.env`. However, the client's instance would be set in a `client/.env` file or by prefixing the run command with the variable, a la:
```sh
FEATURE_FLAG_SELF_SERVICE=ON yarn run start:ssl
```

This particular file is for running GitHub actions. For actions, we set the environment variables as repository settings. The `workflows/production.yml` file pulls these variables from the repository settings and passes them to the action environments where the client and server are being built and deployed. 

*Ignore the `HD_` prefix. It's a quirk of the Heroku action and the `HD_` is dropped when actually passed to the application.*